source/CommonJobs/CommonJobs.MVC.UI/Content/Evaluations/evaluations.css

```diff
@@ -619,153 +624,532 @@ tr.evaluation-generated td {
+}
+/* line 51, ../../sass/Evaluations/_mixins.scss */
+.evaluation .top-evaluation .top-right .comentarios-generales::-webkit-input-placeholder {
+  font: 14px Helvetica, Helvetica Neue;
+  color: #999;
+}
+/* line 166, ../../sass/Evaluations/_evaluation.scss */
+.evaluation .top-evaluation .top-right .comment-btn {
+  padding: 10px;
+  width: 100px;
+  height: 40px;
 }
-/* line 100, ../../sass/Evaluations/_evaluation.scss */
-.evaluation .comentarios-generales:-ms-input-placeholder {
-  font: 90% Helvetica, Helvetica Neue;
+/* line 174, ../../sass/Evaluations/_evaluation.scss */
+.evaluation .explanatory-text {
+  font-size: 14px;
+  color: #878787;
+  font-style: italic;
 }
-/* line 104, ../../sass/Evaluations/_evaluation.scss */
-.evaluation .comentarios-generales:-moz-placeholder {
-  font: 90% Helvetica, Helvetica Neue;
+/* line 182, ../../sass/Evaluations/_evaluation.scss */
+.evaluation .calification-items {
+  border-bottom: solid 1px #BBB;
 }
-/* line 108, ../../sass/Evaluations/_evaluation.scss */
-.evaluation .comentarios-generales::-moz-placeholder {
-  font: 90% Helvetica, Helvetica Neue;
+/* line 185, ../../sass/Evaluations/_evaluation.scss */
+.evaluation .calification-items th, .evaluation .calification-items td {
+  font-weight: normal;
+  width: 100px;
+  text-align: center;
 }
-/* line 113, ../../sass/Evaluations/_evaluation.scss */
-.evaluation input.calificacion {
-  width: 50%;
+/* line 191, ../../sass/Evaluations/_evaluation.scss */
+.evaluation .calification-items th:first-child,
+.evaluation .calification-items td:first-child {
+  text-align: left;
+  padding-left: 40px !important;
+}
+/* line 199, ../../sass/Evaluations/_evaluation.scss */
+.evaluation .calification-items.table-3-columns th:first-child,
+.evaluation .calification-items.table-3-columns td:first-child {
+  width: -moz-calc(100% - (2 * 100px));
+  width: -webkit-calc(100% - (2 * 100px));
+  width: calc(100% - (2 * 100px));
+}
+/* line 207, ../../sass/Evaluations/_evaluation.scss */
+.evaluation .calification-items.hide-column-2 th:nth-child(2) span,
+.evaluation .calification-items.hide-column-2 td:nth-child(2) span {
+  display: none;
 }
-/* line 117, ../../sass/Evaluations/_evaluation.scss */
-.evaluation th.evaluadores:first-child {
-  border-left: none;
+/* line 210, ../../sass/Evaluations/_evaluation.scss */
+.evaluation .calification-items.hide-column-2 th:nth-child(2) .icon,
+.evaluation .calification-items.hide-column-2 td:nth-child(2) .icon {
+  background-image: url(../Images/Evaluations/11-02.png);
 }
-/* line 121, ../../sass/Evaluations/_evaluation.scss */
-.evaluation th.evaluadores {
+/* line 212, ../../sass/Evaluations/_evaluation.scss */
+.evaluation .calification-items.hide-column-2 th:nth-child(2) .icon:after,
+.evaluation .calification-items.hide-column-2 td:nth-child(2) .icon:after {
+  content: "Mostrar";
+}
+/* line 199, ../../sass/Evaluations/_evaluation.scss */
+.evaluation .calification-items.table-4-columns th:first-child,
```

Comment:
I am a CSS neophyte, but I am seeing code that looks duplicated and very specific, maybe it is enough to add a class to these columns.